packages/xrpl/src/Wallet/index.ts

```diff
@@ -32,6 +33,7 @@ import { rfc1751MnemonicToKey } from './rfc1751'
 
 const DEFAULT_ALGORITHM: ECDSA = ECDSA.ed25519
 const DEFAULT_DERIVATION_PATH = "m/44'/144'/0'/0/0"
+const TRAILING_ZEROS_REGEX = /.*\..*[0]+$/u
```

Comment:
Is a more optimized REGEX for this?  The checker seems to hate it

Reply:
You may be able to use a look-behind to check for the decimal place instead of `.*` - https://www.regular-expressions.info/lookaround.html

That combined with your `[0-9]` check may fix the issue?